README.md

```diff
@@ -85,10 +76,10 @@ Thanks a bunch for wanting to help Zooniverse. Here are few quick guidelines to
 0. Submit a Pull Request
 0. Wait for feedback or a merge!
 
-Your Pull Request will run on [travis-ci](https://travis-ci.org/zooniverse/Panoptes), and we'll probably wait for it to pass on MRI Ruby 2.2.1 and JRuby 1.7.18 before we take a look at it.
+Your Pull Request will run on [travis-ci](https://travis-ci.org/zooniverse/Panoptes), and we'll probably wait for it to pass on MRI Ruby 2.3 and JRuby 1.7.18 before we take a look at it.
```

Comment:
we're not using jruby any more...this can go. check this file for details on the [test matrix file](https://github.com/zooniverse/Panoptes/blob/9ed465843998358a9bfdaa10d3779afb58b7036a/.travis.yml)